docs/questions/visualizations/table.md

```diff
@@ -2,11 +2,16 @@
 title: Tables
 redirect_from:
   - /docs/latest/questions/sharing/visualizations/table
+  - /learn/basics/visualizing-data/table
+  - /learn/visualization/table
+  - /learn/metabase-basics/querying-and-dashboards/visualization/table
 ---
 
 # Tables
 
-The **Table** option is good for looking at tabular data (duh), or for lists of things like users or orders. The visualization options for tables allow you to add, hide, or rearrange fields in the table you're looking at, as well as modify their formatting. Check out [Everything you can do with the table visualization](https://www.metabase.com/learn/metabase-basics/querying-and-dashboards/visualization/table).
+![A table in Metabase](../images/table.png)
+
+Tables are data's natural habitat.
 
 ## Rearranging, adding, and removing columns
```

Comment:
I know it says "remove" in the product, but I'd really prefer if we used "hide" in H2 (because of the blockqoute note in this section)